.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Linting python code
+
+on:
+  push:
```

Comment:
Please limit the `push` trigger to the `master` branch. Otherwise for each PR, the CI will be triggered twice (pushing to the branch + creating the PR)

Reply:
Added, thanks.